Makefile

```diff
@@ -1,23 +1,29 @@
 DEV_ROCKS=busted luacov luacov-coveralls luacheck ldoc
 BUSTED_ARGS ?= -v -o gtest
 CASSANDRA ?= 3.10
+PROD_ROCKFILE = $(shell ls lua-cassandra-*.rockspec | grep -v dev)
+DEV_ROCKFILE = $(shell ls lua-cassandra-*.rockspec | grep dev)
+FLAGS ?=
 
 .PHONY: install dev busted prove test clean coverage lint doc
 
 install:
-	@luarocks make
+	@luarocks make $(PROD_ROCKFILE)
 
-dev: install
+install-dev:
+	@luarocks $(FLAGS) make $(DEV_ROCKFILE) OPENSSL_DIR=$(OPENSSL_DIR)
```

Comment:
I believe you can already specify this with `OPENSSL_DIR=/usr/local/opt/openssl/ make dev`

Reply:
nope :( luarocks ignores it

```
~>  OPENSSL_DIR=/usr/local/opt/openssl luarocks install --local luasec
Warning: falling back to curl - install luasec to get native HTTPS support
Installing https://luarocks.org/luasec-0.7alpha-1.rockspec...
Using https://luarocks.org/luasec-0.7alpha-1.rockspec... switching to 'build' mode

Error: Could not find header file for OPENSSL
  No file openssl/ssl.h in /usr/local/include
  No file openssl/ssl.h in /usr/include
You may have to install OPENSSL in your system and/or pass OPENSSL_DIR or OPENSSL_INCDIR to the luarocks command.
Example: luarocks install luasec OPENSSL_DIR=/usr/local
```

Reply:
I'd rather we don't add this new rule to the makefile at all. I'm not sure I understand why we need it.

The Makefile is there only to run `make dev` and install the dev dependencies. There isn't a need to install the library in the LuaRocks tree - one should use `luarocks install lua-cassandra` or `luarocks make` or `make install` for that, but that's it. The tests will already run against the `src` tree.